src/types/Server.tsx

```diff
@@ -244,6 +243,10 @@ export enum ServerAction {
   WS_REST_CLIENT_CONNECTION_OPENED = 'WSRestClientConnectionOpened',
   WS_REST_CLIENT_CONNECTION_ERROR = 'WSRestClientConnectionError',
 
+  BOOT_NOTIFICATION = 'BootNotification',
+
+  METER_VALUES = 'MeterValues',
```

Comment:
METERVALUES is used ?
It's the same as METER_VALUES

Reply:
@RizzoMichael have mis-merged with latest master-qa, I've cleaned up the mess in that branch with him.